cli/Cargo.toml

```diff
@@ -74,6 +74,7 @@ import_map = "=0.9.0"
 jsonc-parser = { version = "=0.19.0", features = ["serde"] }
 libc = "=0.2.121"
 log = { version = "=0.4.14", features = ["serde"] }
+mitata = '=0.0.5'
```

Comment:
I checked the source code for this crate and it seems there's quite a lot of overlap with code that's already present in Deno (coloring, `BenchReporter`). It's not a big deal but I would suggest to directly write that source code in `cli/tools/bench.rs`, then we could get rid of some of the intermediate abstractions like `BenchReporter`. 

Unless you're planning on developing the crate further and want to potentially introduce more features to `deno bench` in the future? TIOLI

Reply:
It's cli and rust port of https://npmjs.com/mitata, but I can move it into deno repo in a follow-up pr.